flang/lib/Lower/OpenMP.cpp

```diff
@@ -125,6 +125,189 @@ createBodyOfOp(Op &op, Fortran::lower::AbstractConverter &converter,
     privatiseVars(converter, *clauses);
 }
 
+static std::int64_t
+createOrderedDependSourceOp(Fortran::lower::AbstractConverter &converter,
+                            Fortran::lower::pft::Evaluation &eval,
+                            SmallVectorImpl<Value> &operands) {
+  auto &firOpBuilder = converter.getFirOpBuilder();
+  auto currentLocation = converter.getCurrentLocation();
+  // The number of associated loops in the doacross loop nest.
+  std::int64_t orderedLevel{0};
```

Comment:
nit: Use LLVM style guidelines in Lower. e.g., `orderedLevel = 0;`

Reply:
Done.